src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,27 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
+const initialState = [];
+
+export const addBook = (bookObj) => ({
+  type: ADD,
+  payload: bookObj,
+});
+
+export const removeBook = (index) => ({
+  type: REMOVE,
+  payload: index,
+});
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD:
+      return [...state, action.payload];
+    case REMOVE:
+      return state.filter((book) => book.id !== action.payload);
+    default:
+      return state;
+  }
+};
+
+export default bookReducer;
```

Comment:
- Good job done so far, but according to the project requirement, you are to use the [createSlice](https://redux-toolkit.js.org/api/createSlice) function from the redux toolkit. Doing this will also help you do away with a lot of boilerplate code and make your code concise and easier to read. Please fix this 👍 👍